bundle.Dockerfile

```diff
@@ -7,7 +7,7 @@ LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=operator-certification-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
 LABEL operators.operatorframework.io.bundle.channel.default.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.14.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.13.0+git
```

Comment:
Not sure why this keeps happening. Obviously we want to stick with 1.14. I see this locally as well.

Reply:
Another automatic change. We may need to align operator-sdk versions I believe.

Reply:
Change was made this code snippet is outdated now.